<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="numLabel">
         <property name="text">
          <string>数    量</string>
         </property>
         <property name="buddy">
          <cstring>lineEditNum</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnCalc">
         <property name="text">
          <string>计      算</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditNum">
         <property name="text">
          <string>12</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="totalLabel">
         <property name="text">
          <string>总    价</string>
         </property>
         <property name="buddy">
          <cstring>lineEditTotal</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="priceLabel">
         <property name="text">
          <string>单    价</string>
         </property>
         <property name="buddy">
          <cstring>lineEditPrice</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEditTotal"/>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEditPrice"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>4</number>
       </property>
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDec"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditBin"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnBin">
         <property name="text">
          <string>进 制 转 换</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnDec">
         <property name="text">
          <string>进 制 转 换</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditHex"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnHex">
         <property name="text">
          <string>进 制 转 换</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="decLabel">
         <property name="text">
          <string>十  进  制</string>
         </property>
         <property name="buddy">
          <cstring>lineEditDec</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="binLabel">
         <property name="text">
          <string>二  进  制</string>
         </property>
         <property name="buddy">
          <cstring>lineEditBin</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="hexLabel">
         <property name="text">
          <string>十六 进 制</string>
         </property>
         <property name="buddy">
          <cstring>lineEditHex</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
